src/ripple/rpc/tests/Subscribe.test.cpp

```diff
@@ -249,10 +249,28 @@ class Subscribe_test : public beast::unit_test::suite
         expect(jv[jss::status] == "success");
     }
 
+    static
+    std::unique_ptr<Config>
+    makeValidatorConfig()
+    {
+        auto p = std::make_unique<Config>();
+        setupConfigForUnitTests(*p);
+
+        // If the config has valid validation keys then we run as a validator.
+        auto const seed = parseBase58<Seed>("shUwVw52ofnCUX5m7kPTKzJdr4HEH");
+        if (!seed)
+            Throw<std::runtime_error> ("Invalid seed specified");
+        p->VALIDATION_PRIV = generateSecretKey (KeyType::secp256k1, *seed);
+        p->VALIDATION_PUB =
```

Comment:
If you're storing base58 encoded strings why not just store the public and private key as strings this way we don't have to pay for an expensive secp256k operation every time we produce the config.